cirq-core/cirq/json_resolver_cache.py

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Methods for caching json serialization."""
```

Comment:
I think the important part is the resolving and not the caching

"""Methods for resolving JSON types during deserialization"""

Reply:
Changed.